docs/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -44,8 +44,7 @@
 \toprule
 \textbf{Date} & \textbf{Developer(s)} & \textbf{Change}\\
 \midrule
-Date1 & Name(s) & Description of changes\\
-Date2 & Name(s) & Description of changes\\
+19/10/23 & Sawyer Tang, Allan Fang, Ryan Yeh & Initial Version of Hazard Analysis\\
```

Comment:
I meant `19/10/23 & Sawyer Tang & Wrote Safety and Security Requirements` like in the Problem Statement/Dev Plan, but I can just change it later when compiling everything.